WooCommerce/Classes/ViewModels/Order Details/Shipping Labels/BoldableTextView.swift

```diff
@@ -0,0 +1,113 @@
+import SwiftUI
+
+/// A text view that takes in a string whose substrings within two asterisks are bolded.
+///
+/// For example, initializing with a string "*Never* will I *stop learning*." will bold the substrings "Never" and "stop learning".
+///
+struct BoldableTextView: View {
+    fileprivate struct Element {
+        let content: String
+        let isBold: Bool
+
+        init(content: String, isBold: Bool) {
+            if isBold {
+                self.content = content.replacingOccurrences(of: boldRegionMark, with: "")
+            } else {
+                self.content = content
+            }
+            self.isBold = isBold
+        }
+    }
+
+    fileprivate static let boldRegionMark = "*"
```

Comment:
Just thinking if it's better to use `**` to mark bold text. It would be similar to markdown and won't have problems if someone decides to use `*` inside the localisation string for other purposes.

Reply:
I agree, `**` matches the markdown syntax more and allows usage of single matching asterisk like to highlight some substring. Updated in: https://github.com/woocommerce/woocommerce-ios/pull/3285/commits/78d1ea47625e0becf31e7242d76036c35516120d